test/errors/scope-resolution/use-shadows-formal1.1-0.good

```diff
@@ -0,0 +1,5 @@
+use-shadows-formal1.chpl:13: In function 'foo':
```

Comment:
Is the unusual name of this `.good` and its detailed twin, with the `-0` suffix, intentional?